pysradb/cli.py

```diff
@@ -34,15 +38,23 @@ def error(self, message):
 
 def _print_save_df(df, saveto=None):
     if saveto:
-        df.to_csv(saveto, index=False, header=True, sep="\t")
+        if saveto.split(".")[-1].strip().lower() == "csv":
```

Comment:
You could simplify this to 
```suggestion
        if saveto.lower().endswith("csv"):
```